CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ ExternalProject_Add(
        -DBENCHMARK_NAMESPACE:STRING=google
```

Comment:
could you please delete this line, it's obsolete and produces a warning during compilation?